packages/react-devtools/get-commit-list.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const {execSync} = require('child_process');
+const readline = require('readline');
+
+const PULL_REQUEST_BASE_URL = 'https://github.com/facebook/react/pull/';
+
+const hashes = [];
+const lines = execSync(`
+  git log --max-count=5 --topo-order --pretty=format:'%H:::%s:::%as' HEAD -- packages/react-devtools-core/package.json
```

Comment:
so these are just the commits that affected the package.json right?

Reply:
Yeah. Sometimes these package JSONs get updated with new dependencies, but they also always get updated as part of a release (to bump the version number).

If we formalize on a tagging process during release, I could switch the script over to use that instead. For now this seemed easier.